odh/base/monitoring/overrides/prometheus-operator/base/servicemonitor.yaml

```diff
@@ -0,0 +1,18 @@
+---
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: odhservicemonitor
```

Comment:
is it possible to insert hypens or something in here?

Reply:
this overrides a resource of the same name from [here](https://github.com/opendatahub-io/odh-manifests/blob/master/prometheus/operator/base/servicemonitor.yaml)